### DISCUSSION TITLE
Toast UI Editor VS Tiptap VS ETC...

### DISCUSSION THREAD

**Discussion by arch-spatula** [react-challengers/Codefolio#198] [Polls]
# 주요 요구사항

1. 성능
    - 번들사이즈
    - 동작여부
3. 마크다운 문법
4. 이미지 업로드 지원

# Toast UI Editor 유지

## 장점

- 리팩토링할 필요 없음
- 이미지 업로드 가능

## 단점

- SSR 지원이 없어 성능이 나쁨(핵심 기능 중 하나)

# Tiptap 변경

## 장점

- SSR 지원

## 단점

- 리팩토링비용
- 이미지 업로드 어려움

# Quill.js

- CSR만 지원하기 때문에 부적절

### quilljs 체험

https://cloverhearts.github.io/quilljs-markdown/

https://codesandbox.io/s/react-quill-with-functional-component-68zcnt?file=/src/index.js

# UIW.js

https://uiwjs.github.io/react-md-editor/

**Comment by Jeremy-Kr** [comment]
최종 투표 결과 UIW.js가 2표로 우세를 차지하였습니다.
주말간 UIW.js에 대한 테스트 브랜치 작성 후
월요일 테스트 진행하고 의사결정 마무리 짓도록 하겠습니다.
